package.json

```diff
@@ -1,6 +1,26 @@
 {
   "name": "homeday-blocks",
   "version": "2.5.0",
+  "author": "Homeday - Frontend Team",
```

Comment:
Maybe make this guy public: https://github.com/orgs/homeday-de/teams/frontend-engineering
Or we can use https://docs.npmjs.com/files/package.json#people-fields-author-contributors and add the frontend team to the list.

Reply:
Or
![image](https://user-images.githubusercontent.com/1734873/67286409-a54edb80-f4d9-11e9-987f-a1e238f241a9.png)

Reply:
Thanks for pointing it out. I've removed the author from `package.json` and added AUTHORS file. 

There is also a helper for git, what's missing, which is also helpful (did not know about it)
https://github.com/homeday-de/homeday-blocks/community